editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/creators/AbstractTextFieldParameterPropertyUICreator.java

```diff
@@ -59,23 +58,20 @@ protected int createTextStyle() {
 	 * @return A Validator checking non-emptiness of mandatory field
 	 */
 	protected IValidator createValidator() {
-		return new IValidator() {
-
-			@Override
-			public IStatus validate(Object value) {
-				if (PropertiesUtils.isRequired(parameter) && (value == null || value.toString().trim().length() < 1)) {
-					return ValidationStatus.error("Parameter " + parameter.getName() + " is a mandatory field and cannot be empty.");
-				}
-				return ValidationStatus.ok();
+		return value -> {
+			if (PropertiesUtils.isRequired(parameter) && (value == null || value.toString().trim().length() < 1)) {
```

Comment:
it looks alike the same code than in AbstractComboFieldParameterPropertyUICreator.createValidator
if it is really the same, it might be worthy to extract a class to factorize it